entries/a/finary.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Finary": {
+    "domain": "finary.com",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "banking",
```

Comment:
Is it really a bank? Seems like it's just portfolio tracking.
```suggestion
```

Reply:
indeed, it is only a portfolio. I added `banking` because it looks like they intend to become a real bank, but this won't be a reality before long